### PR TITLE
Remove lint target in favor of GithubActions workflow

### PR DESCRIPTION
**Pull request by OskarStark**
Closes https://github.com/sonata-project/SonataAdminBundle/pull/5883 (Proof PR)

### CODE REVIEW

File: project/.github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        name: composer-normalize
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2
+
+            - name: "Run composer-normalize"
+              uses: docker://localheinz/composer-normalize-action:0.5.2
```
Comment (core23) [inline project/.github/workflows/lint.yml]: The action was moved: https://github.com/ergebnis/composer-normalize-action

```suggestion
              uses: docker://ergebnis/composer-normalize-action:0.5.2
```

Comment (OskarStark) [reply project/.github/workflows/lint.yml]: @localheinz?

Comment (OskarStark) [reply project/.github/workflows/lint.yml]: Ah now, the name of the action itself does not change, he couldn't rename it if I am right

File: project/.github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2
+
+            - name: "Install Ruby 2.6"
+              uses: actions/setup-ruby@v1
+              with:
+                  ruby-version: '2.6' # Version range or exact version of a Ruby version to use, using semvers version range syntax.
```
Comment (greg0ire) [inline project/.github/workflows/lint.yml]: "semvers"?

Comment (OskarStark) [reply project/.github/workflows/lint.yml]: Copied from the official GitHub actions source 😛 can you add and commit a suggestion please? Thanks

Comment (greg0ire) [reply project/.github/workflows/lint.yml]: I would get rid of the comment entirely, we are not writing a tutorial :P

```suggestion
                  ruby-version: '2.6'
```

**Comment by OskarStark** [comment]
Can you merge and deploy @greg0ire ?

**Pull request merged**